CHANGELOG.adoc

```diff
@@ -38,6 +38,7 @@ Such calls will not work anymore since `null` will not be an acceptable value an
 === Dependency update
 
 - [form] Switch to lexical 0.8.1
+- https://github.com/eclipse-sirius/sirius-components/issues/1936[#1936] [releng] Switch to cypress 12.10.0
```

Comment:
We will wait after 2023.4.0 is released to merge this PR, so before merging this will need to be moved into the (to be created) "2023.6.0 (unreleased)" section of the CHANGELOG.